HelpSource/Classes/Pclump.schelp

```diff
@@ -0,0 +1,76 @@
+TITLE:: Pclump
+summary:: Clumps the pattern into arrays of size n.
```

Comment:
suggestion: A pattern that takes another pattern and groups its values into arrays

Reply:
(you don't clump the pattern (the pattern is just an algorithmic plan), but its values)